ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/error/ErrorInterceptorSpec.scala

```diff
@@ -272,7 +264,7 @@ object ErrorInterceptorSpec {
     override def fails(request: HelloRequest): Future[HelloResponse] = ??? // not used in this test
   }
 
-  /** @param useSelfService       - whether to use self service error codes or a "rouge" exception
+  /** @param useSelfService      - whether to use self service error codes or a "rogue" exception
```

Comment:
well spotted

Reply:
Credit goes to @tudor-da 
https://github.com/digital-asset/daml/pull/11965#discussion_r763875299